app/geo/gui/src/ts/twxplore/gui/geo/states/map/FeaturesByType.ts

```diff
@@ -1,8 +1,10 @@
 import {MapFeature} from "./MapFeature";
 import {MapFeatureTypeState} from "./MapFeatureTypeState";
+import {FilterStateTypes} from "./MapFilterStateTypes";
 
 export interface FeaturesByType {
   features: MapFeature[];
   dirty: boolean;
   featureTypeState: MapFeatureTypeState;
+  attributesState: {[attribute: string]: FilterStateTypes};
```

Comment:
FilterStateTypes is really AttributeState?

Reply:
or MapFeatureAttributeState. Please be consistent and specific about name prefixes. "Attribute" could be anything.